package.json

```diff
@@ -4,7 +4,7 @@
   "description": "Quick and simple deploy tool to host Native Web Applications and Static Web Pages in IPFS",
   "bin": "bin.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "standard"
```

Comment:
following the style of other js projects, we do `"lint": "standard"` and reserve the test keyword for tests.

for pre-commit hook, we use [pre-commit](https://www.npmjs.com/package/pre-commit) and add to the package.json

```JavaScript
pre-commit: [
  "lint",
  "test",  // <- if any
  "coverage"  // <- if any
]
```